hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/HoodieParquetRealtimeInputFormat.java

```diff
@@ -80,58 +77,6 @@ protected HoodieDefaultTimeline filterInstantsTimeline(HoodieDefaultTimeline tim
     return timeline;
   }
 
-  /**
```

Comment:
This is simply moving some code into the helper class HoodieRealtimeInputFormatUtils. This has no logic change so hopefully it should not conflict with your changes. 

Still, if you want me to revert these changes, do let me know.

Reply:
I agree but knowing where I made changes to the same methods and consolidated in a different place due to other requirements :) it would make merging easier if I this is not done here. Hope, this is ok.

Reply:
Ok. Reverted these changed.